apps/AEPSampleAppNewArchEnabled/README.md

```diff
@@ -1,50 +1,93 @@
-# Welcome to your Expo app 👋
+# Welcome to Your Expo AEPSample App with New Architecture Enabled 👍
```

Comment:
Welcome to the Expo AEPSample App with New Architecture Enabled!

Let remove the emoji.

Reply:
removed